src/configs/git_status.rs

```diff
@@ -43,6 +44,7 @@ impl<'a> Default for GitStatusConfig<'a> {
             modified: "!",
             staged: "+",
             untracked: "?",
+            typechange: "⇢",
```

Comment:
For consistency, I think I would prefer calling this `typechanged`. Please also disable it by default by replacing the default to an empty string or `None` to avoid confusing users with additional symbols.